server_admin/topics/authentication/x509.adoc

```diff
@@ -378,6 +380,7 @@ All certificates in trusted-ca-list-for-client-auth.pem must be added to link:{i
 
 {project_name} does not have built-in support for other reverse proxy implementations. However, you can make other reverse proxies behave in a similar way to `apache` or `haproxy`. If none of these work, create your implementation of the `org.keycloak.services.x509.X509ClientCertificateLookupFactory` and `org.keycloak.services.x509.X509ClientCertificateLookup` providers. See the link:{developerguide_link}[{developerguide_name}] for details on how to add your provider.
 
+ifeval::["{kc_dist}" == "wildfly"]
```

Comment:
Except for the `Dumping Headers` section, it should be possible to enable logging and follow the troubleshooting steps.

Reply:
I think troubleshooting and stuff like that should be covered by the new guides, not in these old docs

Reply:
We should rather add a troubleshooting guide in the new server guides, and link to that. I would consider that as a post-17 follow-up task though